src/components/Table/ExpandedTransactionRow.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent } from 'react';
```

Comment:
TODO: delete this file, move `BLOCK_EXPLORERS` to constants and keep the parsing logic someplace